emva1288/process/plotting.py

```diff
@@ -491,8 +491,8 @@ def plot(self, test):
                 label='Data',
                 gid='%d:data' % test.id)
 
-        ax.axhline(test.DSNU1288_DN(),
-                   label='$DSNU_{1288.DN}$',
+        ax.axhline(test.DSNU1288,
+                   label='$DSNU_{1288}$',
```

Comment:
The y-axis units are DN, but `test.DSNU1288` units are $e^-$, so there is a mismatch.

Reply:
Unresolving until all the units actually match.

Reply:
v0.6.3:
![image](https://user-images.githubusercontent.com/862398/187273769-e159496c-6133-440e-8ea5-136931816442.png)

This PR:
![image](https://user-images.githubusercontent.com/862398/187273892-a5f22f59-f760-4a51-940c-af7dcd56ff67.png)

Reply:
Now i see the problem, thanks for this figure

Reply:
I double-checked the 4.0 version standard and I just realized a mistake I made, the DSNU_DN is still useful and I shouldn't remove it, since it's still useful in the plotting. (we just don't report the value anymore but convert it to % and plot figures still rely on that) So I will roll back that part.

Reply:
Figure 12 on Page 31 of the standard is still using $DSNU_{1288.DN}$ for these plots.

Is it possible that this method is not actually deprecated, but just the reporting of it has changed?

Reply:
Heh. We posted at almost exactly the same time. Looks like we agree :smile:

Reply:
yes! I missed one sentence on the 4.0 document, and now everything make sense now.